src/Bicep.Core/Syntax/TestDeclarationSyntax.cs

```diff
@@ -53,9 +53,31 @@ this.Value switch
                 // blocked by assert in the constructor
                 _ => throw new NotImplementedException($"Unexpected type of test value '{this.Value.GetType().Name}'.")
             };
+        
+        public ObjectSyntax? TryGetParameters(ObjectPropertySyntax property)=>
```

Comment:
Do you need to call this from outside this class? If it is only being called from within the `GetParameters` method, it would make more sense for this to be `private`.

Reply:
Following your other comment, I left TryGetParameters as the only way to access the parameters. So, should I leave this being public?